### DISCUSSION TITLE
Example with Emoji Picker

### DISCUSSION THREAD

**Discussion by piszczu4** [ueberdosis/tiptap#4231] [General Questions]
Hi, is there any example which shows how to implement emoji picker? In official documentiation, there is just a single example with inserting zap icon.

**Comment by chaot1xMD** [comment]
Hi,

In extension page under ["Usage"](https://tiptap.dev/api/nodes/emoji#usage) there are tabs with example implementation with autocomplete see EmojiList.jsx and suggestion.js

**Reply by piszczu4** [reply]
That's what i'm talking about. Its Simple example with zap icon and autocomplete. Im looking for an example with icon locker.

**Reply by chaot1xMD** [reply]
![image](https://github.com/ueberdosis/tiptap/assets/36856151/d6f60547-3fad-41d8-927e-2d5b5bc79dfa)
You can use this example to make "locker" with all available icons, you can access all available icons with `editor.storage.emoji.emojis`

**Reply by piszczu4** [reply]
I was thinking about sth like this
https://www.npmjs.com/package/emoji-picker-react
So that user can open it with a button and browse incons instead of using suggestions `:`

**Reply by chaot1xMD** [reply]
Yea, I understand.
Did a quick search and did not find anything like that implemented, but you could try to implement something like it yourself, emojis in editor storage are already grouped and everything so you just need to implement functionality to display them which shouldn't be so difficult.

Cheers.
